content/summer2021/_index.fr.md

```diff
@@ -0,0 +1,535 @@
+---
+title: "Summer 2021"
+date: Wed Jun 16 21:52:01 EDT 2021
+description: "Details, notes, and questions from the Summer 2021 Reading Group."
+draft: true
+---
+
+## Intelligent Distributed Systems Reading Group
+The inaugural Intelligent Distributed Systems Reading Group (IDSRG) will run from June 2021 through September 2021. Our goal is to survey the distributed systems literature with an eye towards thinking of how machine learning and data analysis might be used to make these systems more flexible.
```

Comment:
On peut traduire en francais? Just the sentence on line 9, not the entire page 😉